src/components/TradeWidget/index.tsx

```diff
@@ -719,7 +719,7 @@ const TradeWidget: React.FC = () => {
                   price,
                   priceInverse: invertPriceFromString(price),
                   validFrom: formatTimeToFromBatch(validFrom, 'TIME').toString(),
```

Comment:
Why this `validFrom` doesn't need the fix, but the one at 680 does?

Also, a nitpick would be to create a const with a name like `const isAsap = validFrom === 0` and `doesNotExpire = validUntil === 0` so it's more clear the semantics we are using and it's easier to read

Reply:
because the if block above - ok will change it